src/main/java/hudson/plugins/octopusdeploy/OctopusDeployReleaseRecorder.java

```diff
@@ -409,20 +409,39 @@ private String getReleaseNotesFromScm(AbstractBuild build) {
         if (currentBuild != null) {
             while (currentBuild != build)
             {
-                ChangeLogSet<? extends ChangeLogSet.Entry> changeSet = currentBuild.getChangeSet();
-                for(Object item : changeSet.getItems())
-                {
-                    ChangeLogSet.Entry entry = (ChangeLogSet.Entry)item;
-                    notes.append(entry.getMsg()).append("\n");
-                }
+                String currBuildNotes = convertChangeSetToString(currentBuild);
+                if (!currBuildNotes.isEmpty())
```

Comment:
Just for sake of consistency and readability, please include the curly brackets even with one liners.